apps/web/app/(authenticated)/(app)/app/settings/billing/plans/page.tsx

```diff
@@ -134,7 +134,11 @@ export default async function Page() {
               {tiers[tier].footnotes && (
                 <ul className="mt-6 mb-8">
                   {tiers[tier].footnotes.map((footnote, i) => (
-                    <li key={`note-${i}`} className="flex text-xs text-content-subtle gap-x-3">
+                    <li
+                      // biome-ignore lint/suspicious/noArrayIndexKey: I got nothing better right now
```

Comment:
Used the same explanation (😅) for index key mapping elements as found in other places in the codebase.